src/unreal_plugin/Source/UnrealDeadlineCloudService/Private/DeadlineCloudJobSettings/DeadlineCloudEnvironment.cpp

```diff
@@ -0,0 +1,125 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+
+#include "DeadlineCloudJobSettings/DeadlineCloudEnvironment.h"
+#include "PythonAPILibraries/PythonYamlLibrary.h"
+#include "PythonAPILibraries/PythonParametersConsistencyChecker.h"
+
+UDeadlineCloudEnvironment::UDeadlineCloudEnvironment()
+{
+}
+
+void UDeadlineCloudEnvironment::OpenEnvFile(const FString& Path)
+{
+    if (auto Library = UPythonYamlLibrary::Get())
+    {
+        FEnvironmentStruct EnvironmentStructure = Library->OpenEnvFile(Path);
+        Name = EnvironmentStructure.Name;
+        Variables.Variables.Empty();
+        for (FEnvVariable Variable : EnvironmentStructure.Variables)
+        {
+            Variables.Variables.Add(Variable.Name, Variable.Value);
+        }
+    }
+    else
+    {
+        UE_LOG(LogTemp, Error, TEXT("Error get PythonYamlLibrary"));
+    }
+}
+
+FParametersConsistencyCheckResult UDeadlineCloudEnvironment::CheckEnvironmentVariablesConsistency(UDeadlineCloudEnvironment* Env)
```

Comment:
Could `Env` be a pointer to const?
```suggestion
FParametersConsistencyCheckResult UDeadlineCloudEnvironment::CheckEnvironmentVariablesConsistency(const UDeadlineCloudEnvironment* Env)
```

Reply:
const added